src/localstate.jl

```diff
@@ -0,0 +1,68 @@
+using JuLIP, StaticArrays, LinearAlgebra
+using ACE: BondEnvelope, filter, State, CylindricalBondEnvelope
+
+# function get_atoms(fd::HDF5.File; groupname=nothing)
+#    groupname === nothing && (groupname = HDF5.name(first(fd)))
+#    positions = HDF5.read(fd, string(groupname,"/positions"))
+#    unitcell = HDF5.read(fd, string(groupname,"/unitcell"))
+#    species = HDF5.read(fd, string(groupname,"/species"))
+#    atoms = JuLIP.Atoms(; X = positions, Z = species,
+#                         cell = unitcell,
+#                         pbc = [true, true, true])
+#    return [unitcell, species, positions], atoms
+# end
+# 
+# function get_atoms(fname::String; groupname=nothing)
+#    HDF5.h5open(fname, "r") do fd
+#       get_atoms(fd; groupname=groupname)
+#    end
+# end
+
+```
+get_bond(at,i,j)
+input:  an Atoms at, atomic index i and j
+output: index of the nearest atom having index j, i.e., the one really forms a 
+        bond with atom i
+```
+function get_bond(at::JuLIP.Atoms,i::Int64,j::Int64)
```

Comment:
it appears you are using minimum image convention? Are you using that for TB as well? Is James aware of that? We usually try to avoid this in all of our codes. I'll pick this up in slack.

Even if we have this in TB we cannot use that here since our training sets will always contain some very small unit cells.

Reply:
Yes, it was used in TB code as well. If we want to remove it, does it means that when considering the site energy `E_i` we should consider all possible bonds `i-j_k` instead of the nearest one?

Reply:
yes - this is what we do in JuLIP and also in our very first TB implementation